### PR TITLE
better explain cass/ES/kafka deps and their needed setup

### PR DESCRIPTION
**Pull request by Dieterbe**
@obfuscurity found the docs too vague in explaining what he
needs to do set up the dependencies.
This should clarify.

Also explain some ES gotcha's a bit better.

@obfuscurity how does this look?
@ctdk or @woodsaj is there any cassandra config settings we should recommend? now it just says "use the defaults"

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -67,9 +68,17 @@ This installs only metrictank itself, and none of its dependencies.
...
+For basic setups, you can just install it and start it with default settings.
+To tweak schema and settings, see [Cassandra](https://github.com/raintank/metrictank/blob/master/docs/cassandra.md)
+
+## set up elasticsearch
+
+Also here, you can just install it and start it with default settings. 
+
+## optional: set up kafka
+
+If you want to use a queue, install and start Kafka. Ideally 0.10
```
Comment (obfuscurity) [inline docs/installation.md]: Why would I want or need a queue?

**Comment by ctdk** [comment]
I started working on the docs yesterday, and adding a big section on installing the dependencies. I was working through graphite-metrictank first, but then was going to add sections for cassandra, ES, and kafka (on a basic level, but enough to get going).

**Comment by obfuscurity** [comment]
Generally speaking (this affects other documents), the lack of consistent capitalization and punctuation is triggering my PTSD. I see some level-two headers with lower-case titles and level-three headers with upper-case titles. Numerous sentences missing periods. I'm not trying to pick on your particular writing @Dieterbe, it just so happens I've been reading more of your docs lately because I'm working with metrictank.

@nopzor1200 @mattttt @bulletfactory Do we need a raintank style guide for documentation? As a company encouraging public open source software, these docs are often going to be users' first impressions of our organization. I think it's important that we present ourselves as polished and professional.

**Comment by Dieterbe** [comment]
uppercased all headers. if anyone notices any other obvious typo's or punctuation errors, feel free to commit directly.

**Comment by obfuscurity** [comment]
👍 on the merge but I'd like to continue the discussion around style guidelines.

**Comment by bulletfactory** [comment]
Originally, I didn't think we'd need something like that, since
documentation is not really marketing related (where we'd use something
like "raintank" lowercase). Would some kind of md cheatsheet with common
doc structure for docs help? @obfuscurity Do you have another suggestion on
how we can increase consistency?

I think missing periods are simply typos that would benefit from a
proofread. Some of the title issues may be a carryover from marketing (for
example worlPing, where "w" was always lowercase and "P" was always
capital), but I feel like that doesn't work in documentation or technical
docs. thoughts?

**Pull request merged**